### ISSUE TITLE
Default id parameter present in the Swagger documentation of every route of a @Crud() controller

### ISSUE THREAD

**Issue by MaSydJun** [nestjsx/crud#62]
Hey,

I recently tried the demo project of nestjsx/crud (nestjsx/crud/integration/typeorm).
I recognized that the swagger documentation contains an additional id parameter for every route - even for "get all" or "create one", where it's not necessary to provide such an id at all.

Here is what I mean:
<img width="580" alt="Bildschirmfoto 2019-04-09 um 12 08 52" src="https://user-images.githubusercontent.com/1098870/55792533-23f5a580-5ac1-11e9-9cea-5fbfa60f037c.png">

In the code, the `params` attribute is explicitly set to `{}` (empty).
Is this a bug?

```
// ...
@Crud(Company, {
  params: {},
  routes: {},
  options: {
    join: {
      users: {
        exclude: ['password'],
      },
    },
    sort: [{ field: 'id', order: 'DESC' }],
    maxLimit: 5,
    cache: 10000,
  },
})
@ApiUseTags('companies')
@Controller('companies')
export class CompaniesController {
// ...
```

------

[EDIT]

In `helpers.js` in function `setDefaultCrudOptions()` the parameter `id: number` is used by default:
```
function setDefaultCrudOptions(crudOptions, target) {
    const check = (obj) => shared_utils_1.isNil(obj) || !shared_utils_1.isObject(obj) || !Object.keys(obj).length;
    if (check(crudOptions.params)) {
        crudOptions.params = { id: 'number' };
    }
    // ...
}
```

Is this really necessary? Since not every route has an id parameter.

**Comment by roland-chernov-akvelon** [comment]
Seems this issue was fixed by https://github.com/nestjsx/crud/commit/75418c35d4428d767b507e3237c78a76e7468990 .

**Comment by zMotivat0r** [comment]
Sorry for not closing thing issue on time. Yep it was fixed with v4.0.0
